styles/newsletter.scss

```diff
@@ -30,23 +30,19 @@ $o-grid-gutter-small: oGridGutter(S);
 		display: inline-block;
 		align-self: flex-end;
 		margin: 0;
-
-		// for IE<11
-		max-width: 100%;
+		max-width: fit-content;
 		flex-shrink: 1;
 		padding-top: 20px;
-		// 😢
-		font-size: 30px;
-		line-height: 30px;
+		line-height: 40px;
 
-		@include oGridRespondTo($from: M, $until: XL) {
+		@include oGridRespondTo($from: S) {
 			font-size: 24px;
 			line-height: 28px;
 		}
 
-		@include oGridRespondTo($until: S) {
-			font-size: 40px;
-			line-height: 40px;
+		@include oGridRespondTo($from: M) {
+			font-size: 30px;
+			line-height: 30px;
```

Comment:
Regarding the discussion on the change request with designer, no design modifications were made primarily because they were not included in the scope of this ticket. Consequently, there isn't a specific designer available to consult on this matter, unless we consider reaching out to whoever was responsible for the original design years ago. Nevertheless, I am okay to implementing adjustments that align as closely as possible with the current standards where feasible.